src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+
+const todos = [
+  {
+    _id: 1,
+    title: 'Work',
+    description: 'You have to work to live?',
+    completed: false,
+  },
+  {
+    _id: 2,
+    title: 'Eat',
+    description: 'You have to eat to live?',
+    completed: false,
+  },
+  {
+    _id: 3,
+    title: 'Sleep',
+    description: 'You have to sleep to live?',
+    completed: false,
+  },
+];
+
+const main = () => {
+  const todoList = document.querySelector('.todo-list');
+  for (const todo of todos) {
+    const listItem = document.createElement('li');
+    listItem.classList.add('todo-item');
+    const check = document.createElement('input');
+    check.type = 'checkbox';
+    check.classList.add('todo-checkbox');
+    listItem.innerHTML = `${todo.title}`;
```

Comment:
- In here, after updating the objects, you shouldn't have the `title` in the object again. Also, from the objects in your array, you're only using the `title`, what about the rest properties in the object? let's improve on this.